### DISCUSSION TITLE
New gsplat compression

### DISCUSSION THREAD

**Discussion by CameronFraser** [mkkellogg/GaussianSplats3D#309] [Ideas]
Hello,

There was a recent PR and merge on the gsplat repo that increases compression of splats: https://github.com/nerfstudio-project/gsplat/commit/1dca18461e9dc21a301dd3fc5ad3b2aeb3fc6f20 however I think the output format of the compressed file requires viewers to do on the fly decompression. Just curious if you have had a chance to look at this and if there are any plans to incorporate it. Also curious how it compares to the ksplat format

**Comment by mkkellogg** [comment]
Sorry for the late response on this one, somehow I missed the notification. I actually just recently found out about the addition of the new compression capabilities in nerfstudio, specifically `PngCompression` as mentioned [here](https://docs.gsplat.studio/main/apis/compression.html). It looks like it is quite effective, based on [this comparison](https://w-m.github.io/3dgs-compression-survey), and I'd love to add support for it into my viewer. It's just a matter of finding the time to do it -- hopefully soon!
